### PR TITLE
Implemented Story Download, Location Feed and Pending follow requests

### PR DESCRIPTION
**Pull request by marco2012**
Story download is now fully functional with the simple command `bot.download_stories("username")` as shown in the example folder.

### CODE REVIEW

**Comment by ohld** [comment]
Great! Thank you very much! Can you please fix the TRAVIS-CI codestyle issues? Thanks

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Instabot
+# Instabot with Stories support
```
Comment (ohld) [inline README.md]: I don't think we need to rename the tagline, please remove it from the Pull Request

File: examples/download_stories.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('username', type=str, help='@username')
+args = parser.parse_args()
+if args.username[0] != "@":  # if first character isn't "@"
+    args.username = "@" + args.username
+
+bot = Bot()
+bot.login()
+
+bot.download_stories("") #INSERT USERNAME HERE
```
Comment (ohld) [inline examples/download_stories.py]: username should be taken from command line arguments

File: instabot/api/api_story.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import shutil
+import struct
+import time
+
+from requests_toolbelt import MultipartEncoder
+
+from . import config
+
+def download_story(self, filename, story_url, username):
+    p = "stories/"+username
```
Comment (ohld) [inline instabot/api/api_story.py]: codestyle!

File: examples/comment_medias_by_location.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# coding=utf-8
+w# coding=utf-8
```
Comment (ohld) [inline examples/comment_medias_by_location.py]: ???????????????????

File: README.md (hunk 1 above)
Comment (ohld) [reply README.md]: Please remove it

File: examples/download_stories.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('username', type=str, help='@username')
+args = parser.parse_args()
+if args.username[0] != "@":  # if first character isn't "@"
+    args.username = "@" + args.username
+
+bot = Bot()
+bot.login()
+
+bot.download_stories("")  # INSERT USERNAME HERE
```
Comment (ohld) [inline examples/download_stories.py]: Make this username be inserted by command line arguments, you've already had the code that parses the CLI input

**Comment by marco2012** [comment]
I added support for liking location feed based on a given location with the simple function `bot.like_location_feed("location name", amount)`, as shown in the example folder.

**Comment by marco2012** [comment]
I added support for approving or rejecting pending follow requests for private accounts.

**Comment by ohld** [comment]
Man, I really appreciate your work, but I can not approve your pull request since you've changed the Readme.md file. Please remove changes to this file and I'll approve the PR

**Review by ohld — CHANGES_REQUESTED** [review]
Don't change Readme.md file

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,5 @@
+tests.txt
```
Comment (ohld) [inline .gitignore]: why have you added this to the main repo?

File: instabot/api/api_story.py (hunk 3 above)
Comment (marco2012) [reply instabot/api/api_story.py]: I'm not sure what you mean here, all the tests on codestyle succeed

**Comment by marco2012** [comment]
Everything should be resolved, I switched to the original readme

File: examples/download_stories.py (hunk 2 above)
Comment (marco2012) [reply examples/download_stories.py]: added command line argument

File: README.md (hunk 1 above)
Comment (marco2012) [reply README.md]: switched to original readme

File: examples/comment_medias_by_location.py (hunk 4 above)
Comment (marco2012) [reply examples/comment_medias_by_location.py]: keyboard mistake, removed it

File: examples/download_stories.py (hunk 5 above)
Comment (marco2012) [reply examples/download_stories.py]: added parser in all my examples

**Comment by ohld** [comment]
One last question about `.gitignore`: why have you added a new file to it?

**Comment by marco2012** [comment]
restored original `.gitignore`

**Review by ohld — APPROVED** [review]

**Comment by ohld** [comment]
Thank you very much for your work! I think your new functionality can be very useful for users.

**Pull request merged**